src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.UI/App.xaml.cs

```diff
@@ -30,7 +30,9 @@ protected async override void OnLaunched(LaunchActivatedEventArgs args)
 			.Localization()
 			.Serialization()
 			.Services()
-			.Navigation();
+			.Navigation()
```

Comment:
As part of this PR let's update this to remove all the extra extension methods that we created (as previously discussed)

Reply:
have that in the Template update, I'd rather wait to merge that change

Reply:
Would be better to have it in this PR, or separate PR - I want to get this refactor done before we look at the PR to toggle options in the template to avoid chasing our tails

Reply:
this would cause unnecessary conflicts